docs/user/troubleshooting/03-45-jobs-cant-finish.md

```diff
@@ -5,16 +5,12 @@
 Pods created by Jobs remain stuck in the `NotReady` status after the main containers have finished.
 
 ## Cause
+By default, the Istio module injects `istio-proxy` containers as native sidecars. However, you can also set the annotation `sidecar.istio.io/nativeSidecar` annotation to `"false"` for a specific Pod. This annotation overwrites the default setting and indicates that instead of native sidecars, the annotated Pod must be injected with a regular sidecar container.
```

Comment:
```suggestion
By default, the Istio module injects `istio-proxy` containers as native sidecars. However, you can also set the annotation `sidecar.istio.io/nativeSidecar` to `"false"` for a specific Pod. This annotation overwrites the default setting and indicates that instead of native sidecars, the annotated Pod must be injected with a regular sidecar container.
```